gfx/drivers_shader/shader_gl_cg.c

```diff
@@ -1074,16 +1074,27 @@ static void *gl_cg_init(void *data, const char *path)
 
    memset(cg->alias_define, 0, sizeof(cg->alias_define));
 
-   if (    !string_is_empty(path)
-         && string_is_equal(path_get_extension(path), "cgp"))
    {
-      if (!gl_cg_load_preset(cg, path))
-         goto error;
-   }
-   else
-   {
-      if (!gl_cg_load_plain(cg, path))
-         goto error;
+      bool is_preset;
```

Comment:
C89 compliance issue. Variables need to be declared either at the top of a code block or function.